### PR TITLE
[NALM-313] How-to OTA update

### PR DESCRIPTION
**Pull request by smarki**

### CODE REVIEW

**Comment by ConnorLand** [comment]
I plan on merging this PR into NALM-312, then merging 312 into development.

File: content/howto/mobile/how-to-ota.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+   ![Output of rollback command](attachments/how-to-ota/rollback-result.png)
+
+3. Next time you open your app, you should be greeted with the **Update available** dialog box. Tap **Confirm** to roll your app back on your device.
+
+## 5 Initiating a Partial Initial Rollout
+
+### 5.1 When is it Safe to Rollback an Update?
+
+Please see [When OTA updates are safe to do without redeploying a new Mendix App](#safeToUpdate).
```
Comment (ConnorLand) [inline content/howto/mobile/how-to-ota.md]: Look at this subsection and section above it, and fix it. Change to just text and link.

File: content/howto/mobile/how-to-ota.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+Mendix Native Apps are offline first. This means you should be cautious when changing the following elements, and should avoid changing them if possible: 
+
+* The navigation profile
+* An offline first entity, for example entity name changes, new entity relationships, and more
+
+Generally, you should avoid doing destructive changes to offline-synced entities. In the rare cases this is unavoidable, releasing a new app version or doing over the air updates might put your app's users in an unrecoverable state. 
+
+### 7.1 Offline Apps and Data Loss
+
+Data loss can occur when OTA updates or new releases coincide with apps being offline. For example, imagine your Mendix developers were hard at work optimizing the data store entity structure to speed up sync operations. They are quite happy with their results and release that morning. They push a new runtime by pressing the **Run** button in Studio Pro, and then run the Native Builder to push a new update to the apps. All seems to work fine.
```
Comment (ConnorLand) [inline content/howto/mobile/how-to-ota.md]: Shorten this story, boil down the basic "do not do x,y,z" into bullets

Comment (ConnorLand) [reply content/howto/mobile/how-to-ota.md]: Delineate that while renaming/removing entities is a problem, adding entities is NOT a problem.

File: content/howto/mobile/how-to-ota.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
+---
+title: "Use Over The Air Updates"
+parent: "native-mobile"
+menu_order: 71
+description: "A how to push over the air update (OTA), roll it back or modify it"
+tags: ["native", "mobile", "ota", "native-builder", "over the air"]
+---
+
+## 1 Introduction
```
Comment (ConnorLand) [inline content/howto/mobile/how-to-ota.md]: Stefanos: mention somewhere that it is always best practice to click "Deploy" in Studio Pro after doing X

File: content/howto/mobile/how-to-ota.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+Using Native Builder 3.0 and Mendix Studio 8.4, you can update your Mendix Native Apps over the air (OTA).
+OTA updates are a fast and painless way of updating things like layouts, pages, assets or even you app's business logic (such as nanoflows and JavaScript actions).
```
Comment (ConnorLand) [inline content/howto/mobile/how-to-ota.md]: Stefanos: Danny had some notes about moving "layout" to "changes in pages" here. That will allow you to remove widget, static, and more. I guess just tidy up this list to your satisfaction?

Comment (smarki) [reply content/howto/mobile/how-to-ota.md]: If I remember correctly; he liked the seperation of reasons after all.

**Pull request merged**